src/app/tabs/tabs-angular.demo.html

```diff
@@ -7,7 +7,7 @@
 <div class="clr-example">
     <button class="btn btn-primary" (click)="inOverflow = !inOverflow">Toggle</button>
     <clr-tabs>
-        <clr-tab *ngIf="true">
+        <clr-tab>
```

Comment:
Why is `*ngIf="true"` removed though? It was added there for testing purposes.

Reply:
We have several unit tests for it, so we're safe.

I can put it back if you want, but it just gives you a false sense of security which tricked me once already. I changed some code and suddenly it worked again, only for me to discover that instead of registering the tabs with `*ngIf` last, it just registered them first. So suddenly the broken behavior was with the first tab _without_ `*ngIf`, and the second tab _with_ it.